Dockerfile

```diff
@@ -19,6 +23,7 @@ RUN rm src/*.rs
 # Copy over source files
 # COPY ./tests ./tests
 COPY ./src ./src
+COPY ./migrations ./migrations
```

Comment:
Maybe this should be swapped with line 25? I feel like the source files will be modified more frequently than migrations.

Reply:
Does it really matter?

Reply:
Wait actually where is the database actually set up? I reread the dockerfile and it looks like the database isn't actually created or run from it, so I'm not sure what the migrations are doing in the image. Like unless we're running `diesel migration run` or something in the image, those files never get touched.

Reply:
If we were building the database from the dockerfile, then it would matter if we wanted to do stress testing/benchmarking. Ideally we'd run all the database setup stuff first, then run source copying second, then building third. That way in the common case we don't have to run migrations on a potentially large number of rows.

Reply:
But that's probably not applicable here, bc I'm not sure when the migration files copied in ever get read.

Reply:
If they're meant to be on the container that's used for testing, I think those lines need to be swapped after the second `FROM` command, so that they're included in the final image; I don't know why we'd want to do that though, because from what I can tell the postgres server being build from the `docker-compose` file